backend/tests/unit/apps/owasp/api/internal/mutations/board_candidate_claim_test.py

```
@@ -516,11 +516,14 @@ def test_reorder_claims_mixed_boards(self, mock_claim_model):
 class TestCreateBoardCandidateClaim:
     """Tests for create_board_candidate_claim mutation."""
 
-    def _make_input_data(self, name="Test Claim", description="Test description", year=2025):
+    def _make_input_data(
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_make_input_data`

(ANN202)